runbotics/runbotics-orchestrator-ui/src/src-app/views/auth/RegisterPage/Register.page.tsx

```diff
@@ -97,6 +98,23 @@ const RegisterPage: FC = () => {
     const dispatch = useDispatch();
     const { enqueueSnackbar } = useSnackbar();
 
+    const invitingTenant = useSelector((state) => tenantsSelector(state).invitingTenant);
+
+    useEffect(() => {
+        dispatch(tenantsActions.fetchTenantNameByInviteCode(inviteCodeURL))
+            .unwrap()
+            .catch((error) => {
+                if (error.statusCode === 400) {
```

Comment:
I think it'd great to have universal function for handling error codes -> so if it gets error code 404 it redirects to /403 (and there should be page with "Access denied" message, for >500 we could return unexpected error, for 404 redirect to /404 with message 'not found'.

the function should take error as param and redirect in the background or return string with adequate message

Reply:
I agree with @stun-is-law idea. I'm just not sure if it is the point of this task/should be done in another task?

Reply:
yeah, another task. should be added to to be discussed sprint

Reply:
I'll do it